backend/src/core/middlewares/api-key.middleware.ts

```diff
@@ -4,6 +4,14 @@ import { ApiKeyService } from '@core/services'
 
 const logger = loggerWithLabel(module)
 
+const listApiKeys = async (
+  req: Request,
+  res: Response
+): Promise<Response | void> => {
```

Comment:
actually the `Promise<Response | void>` return signature is a bit weird, but it's a larger problem with the codebase, so might as well just stick with it for now.